src/components/TransactionsTable/TransactionRow.tsx

```diff
@@ -110,6 +114,18 @@ export type TransactionRowProps = {
   onRowClick: (rowId: string) => void;
 };
 
+export type TransactionPaymentReference = {
+  txn: Transaction;
+};
+
+function PaymentReference({txn}: TransactionPaymentReference) {
+  const { t } = useTranslation('transactions');
+  if (txn.cashinDetails?.concepto) {
+    return <RowTextDetail><span>{t('table.reference')}</span>{txn.cashinDetails?.concepto}</RowTextDetail>;
+  }
+  return (null);
```

Comment:
could be simplified something like:
```suggestion
    return {!!txn.cashinDetails?.concepto && (<RowTextDetail><span>{t('table.reference')}</span>{txn.cashinDetails?.concepto}</RowTextDetail>)}
```

Reply:
no funco
![image](https://github.com/user-attachments/assets/2a3058d5-b7e0-4a0d-9334-5f043e17bd41)